tests/cover/test_find.py

```diff
@@ -42,7 +42,7 @@ def test_can_find_nan():
     find(floats(), math.isnan)
 
 
-def test_can_find_nans():
+def test_can_find_NaNs():
```

Comment:
Personal preference again, but I preferred the old way – the convention is for function names to be lower case, and this includes NaN. cf: `math.isnan`, `test_can_find_nan`.